src/server/game/Spells/SpellMgr.cpp

```diff
@@ -1652,6 +1652,7 @@ void SpellMgr::LoadSpellProcs()
     }
 
     isTriggerAura[SPELL_AURA_DUMMY] = true;
+    isTriggerAura[SPELL_AURA_PERIODIC_DUMMY] = true;
```

Comment:
This should not be needed, it is only there to automatically generate SpellProcEntry data based on SpellInfo alone, without using `spell_proc` table at all

Reply:
Right, although it won't automatically generate it, it will only do so if it has proc flags.
Any reason we don't want that?

Without it I need to create a dedicated spell_proc entry, while using the exact same values that are already on the spell.

Reply:
This is actually fine